app/src/main/java/se/rebeccazadig/bokholken/login/UserViewModel.kt

```diff
@@ -34,9 +33,9 @@ class UserViewModel : ViewModel() {
 
         viewModelScope.launch {
             val userid = loginRepo.getUserId()
-            val userInfo = userName.value.orEmpty()
-            val userContact = userContact.value.orEmpty()
-            val userCity = userCity.value.orEmpty()
+            val userInfo = userName.value.toString()
```

Comment:
Changing **.orEmpty** to **.toString** is not gonna solve any problem.
Let's consider that userName is null wrapped inside MutableLiveData.
 **.orEmpty** will convert it to "" (empty string) and later you will need to validate data (meaning that you don't want create a user with "" name).
 **.toString** will return "null" which is a string of 4 letters. So you will create a user som heter "null".
 The same for other fields. You need to decide which fields are mandatory and validate user input for this fields.